### PR TITLE
fix: fixed Xcode 14 build error

### PR DESCRIPTION
**Pull request by CodeEagle**
## Connection with issue(s)

Resolve issue #1216

Connected to #1216

## To Do

- [x] double check the original issue to confirm it is fully satisfied
- [x] add testing notes and screenshots in PR description to help guide reviewers
- [x] request the "UX" team perform a design review (if/when applicable)

### CODE REVIEW

**Review by mikhael28 — APPROVED** [review]
Can confirm personally that this merge fixes this issue: https://github.com/pichillilorenzo/flutter_inappwebview/issues/1216

**Comment by mikhael28** [comment]
@pichillilorenzo would you please review and merge this in? Right now, in the XCode beta, I have to manually edit the package file.

**Comment by MrJohnDev** [comment]
Merge it, Works on XCode 14 Beta 4

File: ios/Classes/Types/UserScript.swift [hunk 1]
```diff
@@ -10,8 +10,19 @@ import WebKit
 
 public class UserScript : WKUserScript {
     var groupName: String?
+
+    var contentWorldWrapper: Any?
     @available(iOS 14.0, *)
-    lazy var contentWorld: WKContentWorld = WKContentWorld.page
+    var contentWorld: WKContentWorld {
+      get {
+        if let value = contentWorldWrapper as? WKContentWorld {
```
Comment (EPik-dev) [inline ios/Classes/Types/UserScript.swift:18]: To keep lazy behaviour i'd suggest the following implementation
```
if contentWorldWrapper == nil {
    contentWorldWrapper = WKContentWorld.page
}
return contentWorldWrapper as! WKContentWorld
```

, but yours is safer

**Review by EPik-dev — APPROVED** [review]

**Comment by sujithv28** [comment]
When will this be merged?

**Comment by Attiv** [comment]
When will this be merged?

**Comment by cimplesid** [comment]
When will this be merged?

**Comment by cimplesid** [comment]
Until it is merged you can use 
`
flutter_inappwebview:
    git: https://github.com/CodeEagle/flutter_inappwebview
`

**Comment by huylethanh** [comment]
Please merge it

**Comment by mikhael28** [comment]
@cimplesid boss

**Comment by j-ixcayau** [comment]
Please merge it, it works for me too

**Comment by masatada-kurihara** [comment]
@pichillilorenzo Please merge it

**Comment by calmog** [comment]
Anyone finding themselves with a conflict between the git dependency and a transitive dependency on the hosted flutter_inappwebview, can use this code in their pubspec.yaml:

`dependency_overrides:
  flutter_inappwebview:
    git:
      url: https://github.com/CodeEagle/flutter_inappwebview`

**Comment by EricksonMoura45** [comment]
> Anyone finding themselves with a conflict between the git dependency and a transitive dependency on the hosted flutter_inappwebview, can use this code in their pubspec.yaml:
> 
> ```
> dependency_overrides:
>   flutter_inappwebview:
>     git:
>       url: https://github.com/CodeEagle/flutter_inappwebview
> ```

Thanks, Buddy :)

**Comment by emrementese** [comment]
> Anyone finding themselves with a conflict between the git dependency and a transitive dependency on the hosted flutter_inappwebview, can use this code in their pubspec.yaml:
> 
> ```
> dependency_overrides:
>   flutter_inappwebview:
>     git:
>       url: https://github.com/CodeEagle/flutter_inappwebview
> ```

Thank you. İts workly  !

**Comment by calmog** [comment]
Any ETA as to when will this fix be merged? RN I need to add the dependency override to allow a build on iOS, but it breaks the build on Android, so I have to constantly go back and forth with adding/removing it.

**Comment by sowmitri123** [comment]
What is the ETA on when this fix be merged?? Currently i am not able build the app for iOS.

**Comment by n13** [comment]
Another workaround: Set deployment target for pods and project to iOS 14.

May or may not be a good workaround for you, just dropping it here.

It's a good one for me, because I need WASM support from inapp_webview, and on iOS the earliest version of iOS that supports WASM in JavaScriptCore is iOS 14. So now I have 2 reasons to require 14 as the minimum version. Previously the app would show an error dialog asking users to upgrade, now it just won't run at all on 13 and below.

**Comment by charlieforward9** [comment]
Please merge this. I would appreciate not implementing in a workaround when the solution exists!

**Comment by Yokoi-K** [comment]
@pichillilorenzo 

This PR fix solved it for us too.
Please MERGE as soon as possible.🙏🙇‍♂️

**Comment by duy-mai-otsv** [comment]
Merge, please.....

**Comment by anggrayudi** [comment]
This is a blocker and should be fixed ASAP.

**Comment by alexzhcn** [comment]
> Until it is merged you can use
> 
> ```
> flutter_inappwebview:
>     git: https://github.com/CodeEagle/flutter_inappwebview
> ```
> 
> and then run the following command
> 
> ```
> flutter pub get && cd ios && pod install
> ```

It works

**Comment by DavidGrunheidt** [comment]
Also blocked a release for my app. Please merge it

**Comment by MagneticRegulus** [comment]
This is also a blocker for an app release for me. We have a small user base, and we are required to support older versions of iOS. Please merge.

Note, we are dependent on this because we use [youtube_player_flutter](https://pub.dev/packages/youtube_player_flutter).

**Review by AlexT84 — APPROVED** [review]

**Comment by 10ndavis** [comment]
@pichillilorenzo let's get this merged!

**Comment by lowapple** [comment]
Plz merge it!

**Comment by trumdongnat** [comment]
Please merge it.

**Comment by pichillilorenzo** [comment]
Released new version `5.4.3+8`, thanks

**Pull request merged**